### PR TITLE
t/155: WebKit support in env utility

### LINKED ISSUE #155: Bring WebKit support to env utility
A requirement of https://github.com/ckeditor/ckeditor5-engine/issues/951.

### PR DESCRIPTION
**Pull request by oleq**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Feature: WebKit support in the env utility. Closes #155.

### CODE REVIEW

File: tests/env.js [hunk 1]
```diff
@@ -19,6 +19,12 @@ describe( 'Env', () => {
 		} );
 	} );
 
+	describe( 'webkit', () => {
+		it( 'is a boolean', () => {
+			expect( env.mac ).to.be.a( 'boolean' );
```
Comment (oskarwrobel) [inline tests/env.js]: `env.webkit`

**Comment by Reinmar** [comment]
I'd certainly separate Blink and Webkit. These engines diverged significantly already.

**Comment by Reinmar** [comment]
Besides, if we can avoid code branching (see my comment under https://github.com/ckeditor/ckeditor5-engine/issues/952) it's worth considering.

**Comment by oleq** [comment]
We may not need this PR after all, see https://github.com/ckeditor/ckeditor5-engine/pull/952#issuecomment-301023131.

**Pull request closed without merging**